adapters/appnexus/appnexus.go

```diff
@@ -89,12 +89,18 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.E
 	// but some SDKs will put it in imp.ext.prebid instead
 	defaultDisplayManagerVer := buildDefaultDisplayManageVer(request)
 
-	var adPodId *bool
+	var (
+		adPodId         *bool
+		uniqueMemberIds []string
+	)
 
 	for i := 0; i < len(request.Imp); i++ {
 		memberId, impAdPodId, err := preprocess(&request.Imp[i], defaultDisplayManagerVer)
 		if memberId != "" {
```

Comment:
We should check for the err here before checking any returned value from preprocess function.

Reply:
nvm. this is addressed in subsequent commit.